bot/cogs/spookysound.py

```diff
@@ -0,0 +1,39 @@
+import random
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+HACKTOBERBOT_VOICE_CHANNEL_ID = 101010  # Replace with actual channel ID
```

Comment:
before we can merge this, this should actually be replaced.

```suggestion
HACKTOBERBOT_VOICE_CHANNEL_ID = 498804789287714816
```

Reply:
Updated.